examples/ex6.cpp

```diff
@@ -152,8 +152,14 @@ int main(int argc, char *argv[])
    //     flux to get an error indicator. We need to supply the space for the
    //     smoothed flux: an (H1)^sdim (i.e., vector-valued) space is used here.
    FiniteElementSpace flux_fespace(&mesh, &fec, sdim);
-   ZienkiewiczZhuEstimator estimator(*integ, x, flux_fespace);
-   estimator.SetAnisotropic();
+   // ZienkiewiczZhuEstimator estimator(*integ, x, flux_fespace);
+   // estimator.SetAnisotropic();
+
+   NewZienkiewiczZhuEstimator estimator(*integ, x);
+   if (dim == 3 && mesh.GetElementType(0) != Element::HEXAHEDRON)
```

Comment:
This may be confusing to users. Would it be better to hide this by doing it inside the constructor for `NewZienkiewiczZhuEstimator`? Since the constructor has a `GridFunction` as an argument, it can access the mesh and check the dimension and element type.

Reply:
Oh. This is modification to `ex6.cpp` not meant to be merged. I just wanted to make it easy for reviewers to test the new estimator. It's not necessary to shift the spectrum when reconstructing on patches of hexahedral elements.